src/components/CalendarHeader.jsx

```diff
@@ -10,13 +10,27 @@ const CalendarHeader = (props) => {
   const { values } = props;
   return (
     <StyledCalendarRow>
-      <StyledCalendarHeader><StyledDayHeaderText>{values[0]}</StyledDayHeaderText></StyledCalendarHeader>
-      <StyledCalendarHeader><StyledDayHeaderText>{values[1]}</StyledDayHeaderText></StyledCalendarHeader>
-      <StyledCalendarHeader><StyledDayHeaderText>{values[2]}</StyledDayHeaderText></StyledCalendarHeader>
-      <StyledCalendarHeader><StyledDayHeaderText>{values[3]}</StyledDayHeaderText></StyledCalendarHeader>
-      <StyledCalendarHeader><StyledDayHeaderText>{values[4]}</StyledDayHeaderText></StyledCalendarHeader>
-      <StyledCalendarHeader><StyledDayHeaderText>{values[5]}</StyledDayHeaderText></StyledCalendarHeader>
-      <StyledCalendarHeader><StyledDayHeaderText>{values[6]}</StyledDayHeaderText></StyledCalendarHeader>
+      <StyledCalendarHeader>
```

Comment:
Possibly cut down some lines here using the map function as well?

Reply:
Thanks again! Will implement this in a future PR.